src/swell/tasks/move_da_restart.py

```diff
@@ -93,8 +93,15 @@ def cycling_restarts(self):
             filename = os.path.basename(filepath).split('.')[0]
             move_files(self.logger, filepath, self.at_next_fcst_dir(filename))
 
-        move_files(self.logger, self.forecast_dir('tile.bin'),
-                   self.at_next_fcst_dir('tile.bin'))
+        # Create a dictionary of src/dst for the single files
+        # ---------------------------------------------------
+        src_dst = {'tile.bin': '',
+                   'RESTART/iced.nc': 'INPUT',
+                   }
+
+        for src, dst in src_dst.items():
+            dst = os.path.join(dst, src.split('/')[-1])
```

Comment:
```suggestion
            dst = os.path.join(dst, os.path.basename(src))
```

Same thing, but communicates intent more clearly.

(Note that _both_ methods will return `''` if `src` has a trailing slash, like `src = "some/path/").